packages/core/test/endowmentsToolkit.spec.js

```diff
@@ -16,12 +16,12 @@ test('getEndowmentsForConfig', (t) => {
       stringValue: 'yabbadabbadoo',
     },
   }
-  const config = {
+  const policy = {
     globals: {
       'namespace.stringValue.includes': true,
     },
   }
-  const resultGlobal = getEndowmentsForConfig(sourceGlobal, config)
+  const resultGlobal = getEndowmentsForConfig(sourceGlobal, policy)
```

Comment:
(suggestion) seems like we should rename `getEndowmentsForConfig` to `getEndowmentsForPolicy`, then alias the former to the latter, and `@deprecate` the former

Reply:
Didn't wanna go that far in this PR. Generally agree.